docker/ssl/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+nginx:
+  build: nginx-image
+  links:
+    - letsencrypt
+    - iri
+  environment:
+    - MY_DOMAIN_NAME=__DOMAIN__
+    - PORT=__PORT__
+  ports:
+    - "80:80"
+    - "443:443"
+  volumes_from:
+    - letsencrypt
+letsencrypt:
+  image: quay.io/letsencrypt/letsencrypt:latest
```

Comment:
I don't really like the idea of pointing to a non-official image for this. Although there are none available. I suggest therefore to pull from a more reputable and long-term image source (https://hub.docker.com/r/linuxserver/letsencrypt/ seems to serve the same functionality) and to point to it by digest, rather than by tag: so we make sure we are always pulling the image we have reviewed with our own eyes, instead of relying on a floating docker image tag.
For example you can pull the image `linuxserver/letsencrypt:170` by its hash: `docker pull linuxserver/letsencrypt@sha256:ab7cafe902eed5b606ebf4b468f826d9cd5846ea0905f844554fd56ad6a78bdc`

Reply:
Sounds good, I agree with your idea of pointing to the digest. However, I believe the quay.io is official. I just realized that they also push to https://hub.docker.com/r/certbot/certbot/ and will be completely transitioning there soon (the only difference is that the one on Docker Hub is based on Alpine Linux). If you want me to switch to that one instead, I can

Reply:
Digest is good enough for me, I will leave to you the choice of following the official EFF one or not.